content/guides/mobile/ios/quick-start/1-create-ios-app.md

```diff
@@ -0,0 +1,63 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Create an iOS App
+
+Before installing the Box iOS SDK you'll need to have an iOS application
+available to add the dependencies into. We'll create a new blank application
+now.
```

Comment:
"We'll create a new blank application now" yet then we give people a choice to either use a blank app or an existing one.

Reply:
Removed the second choice - now just a straightforward guide.